specutils/spectra/spectral_coordinate.py

```diff
@@ -563,7 +572,7 @@ def _project_velocity_and_shift(self, init_vel, fin_vel):
         #  we may end up with a final velocity very close to, but not quite at,
         #  zero. In this case, set a tolerance for the final velocity; if it's
         #  below this tolerance, assume the delta velocity is essentially nan.
-        if np.isnan(delta_vel) or np.abs(np.sum(fin_vel, axis=0)) < 1e-7 * fin_vel.unit:
+        if np.isnan(delta_vel) or np.abs(np.linalg.norm(fin_vel)) < 1e-7 * fin_vel.unit:
```

Comment:
As before, sum doesn't return the vector length

Reply:
Strictly I think the `np.abs` is redundant now, as it is acting on the 2-norm.